### PR TITLE
edited few details

### PR DESCRIPTION
**Pull request by shanmukh9**
edited few details related to Continuous Integration & Continuous Delivery

### CODE REVIEW

**Comment by saintdle** [comment]
The heading changes are correct, line 63 would need updating after the heading changes to say CI not CD.

**Comment by MichaelCade** [comment]
@saintdle what needs to happen here please?

File: Days/day05.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ Once that's in place you go right back to the beginning to the planning stage an
...
-### Continuous Delivery:
+### Continuous Integration:
 
-Continuous Delivery = Plan > Code > Build > Test 
+Continuous Integration = Plan > Code > Build > Test 
 
-### Continuous Integration:
+### Continuous Delivery:
 
 This is effectively the outcome of the Continuous Delivery phases above plus the outcome of the Release phase. This is the case for both failure and success but this is fed back into continuous delivery or moved to Continuous Deployment.
```
Comment (saintdle) [inline Days/day05.md:63]: This needs to say "This is effectively the outcome of the Continuous Integration"

Comment (saintdle) [reply Days/day05.md:63]: @MichaelCade added comment for change, as I can't contribute the edit myself.

**Comment by MichaelCade** [comment]
Hey @shanmukh9 could you make the change and i will merge this PR

**Comment by MichaelCade** [comment]
If this is still relevant then please update and i will merge this week.

**Comment by MichaelCade** [comment]
I have conflicts that must be resolved

**Comment by shanmukh9** [comment]
already made a change, can you merge it?

**Comment by saintdle** [comment]
> already made a change, can you merge it? @MichaelCade

There's still conflicts reported

**Comment by shanmukh9** [comment]
I created separate pull request for this (https://github.com/MichaelCade/90DaysOfDevOps/pull/371) hence closing this

**Pull request closed without merging**